<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioBusManagerWindow</class>
 <widget class="QMainWindow" name="AudioBusManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>526</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="m_addBusPB">
        <property name="text">
         <string>Add bus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_removeBusPB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove bus</string>
        </property>
        <property name="shortcut">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="m_fileNameLabel"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="m_busWidgetsScrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="m_busParentWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>882</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_resetAction"/>
    <addaction name="m_loadAction"/>
    <addaction name="m_saveAction"/>
    <addaction name="m_saveAsAction"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="m_resetAction">
   <property name="text">
    <string>New Layout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="m_loadAction">
   <property name="text">
    <string>Open Layout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="m_saveAction">
   <property name="text">
    <string>Save Layout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="m_saveAsAction">
   <property name="text">
    <string>Save Layout As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="m_removeBusAction">
   <property name="text">
    <string>Remove bus</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="m_addBusAction">
   <property name="text">
    <string>Add bus</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
